sfts/redshift_to_s3.py

```diff
@@ -206,6 +221,12 @@ def object_key_builder(key_prefix, *args):
                    'than end_date: {end_date}.')
 
     object_key = object_key_builder(object_prefix,start_date,end_date)
+
+elif 'date_list' in config:
+    # set dates requested in date_list
+    date_key = "_".join(dates)
```

Comment:
This feels like the appropriate way to name these objects, with the requested dates as part of the object name. That said, if you pass a list of many individual dates in the config file, the object name would be very long.